### ISSUE TITLE
Triangle PS5 demo

### ISSUE THREAD

**Issue by JustAHippo** [InoriRus/Kyty#58]
Could we have the demo for PS5 to test shown in the readme?

**Comment by InoriRus** [comment]
This is sample from PS5 SDK.
If you have official SDK you can compile you own demo.

**Comment by Brutal-sam** [comment]
@JustAHippo the link to download PS5 sdk sample (demo) is in the links of the tutorial https://youtu.be/LDPidynbElk

**Comment by cualquiercosa327** [comment]
Hello, have you seen than ps5 eboot can be already decompiled?
I know than the emu is on a early stage booting ps4 games but they are one which was designed with ps4 sdk/devkit and next port to ps5 (it shares file formats, engine.. with the ps4 version of it precuel) : Astrobot Playroom.
I have some information about it if it can help you

**Comment by Graubek** [comment]
> Hello, have you seen than ps5 eboot can be already decompiled? I know than the emu is on a early stage booting ps4 games but they are one which was designed with ps4 sdk/devkit and next port to ps5 (it shares file formats, engine.. with the ps4 version of it precuel) : Astrobot Playroom. I have some information about it if it can help you

InoriRus has stopped working on the emulator, but there are some emulators that are active: 

- shadPS4: https://github.com/georgemoralis/shadPS4
- fpPS4: https://github.com/red-prig/fpPS4
- obliteration: https://github.com/obhq/obliteration

**Comment by cualquiercosa327** [comment]
It is a shame.
I want give him lot of thanks for their work and I wish him success with his new proyects. 
Do you know if they are someone resuming the proyect?
For now it is the most advanced ps4/5 emulator.
Thanks

**Comment by Graubek** [comment]
> It is a shame. I want give him lot of thanks for their work and I wish him success with his new proyects. Do you know if they are someone resuming the proyect? For now it is the most advanced ps4/5 emulator. Thanks

Currently, no one is continuing to work on Kyty, but I wouldn't be surprised if these emulators would also start supporting PS5 after the PS4 is finished.

**Comment by cualquiercosa327** [comment]
> > It is a shame. I want give him lot of thanks for their work and I wish him success with his new proyects. Do you know if they are someone resuming the proyect? For now it is the most advanced ps4/5 emulator. Thanks
> 
> Currently, no one is continuing to work on Kyty, but I wouldn't be surprised if these emulators would also start supporting PS5 after the PS4 is finished.

I don't believe they need the ps4 support be finished to implement ps5 as they share a lot of things.
In fact seing the change of Sony : they port their ps5 games to pc, i doubt they are many interest on emulate it.
In fact it seems the only which will need an emulator for playing on other devices is the demo Astroplayroom (and maybe psvr2, we still have no information about what Sony will do).
Fun fact : the demo is one of the most interesting (it shows what Dualsense can do), it shares most of things with Astrobot Rescue of psvr1 : Engine, file formats.. and it was build almost it was very advanced with ps4 sdk/devkit less the haptics which were worked in parallel on a pc and finally they paste everything and compile with ps5 sdk.

**Comment by ThreeDeeJay** [comment]
> [@JustAHippo](https://github.com/JustAHippo) the link to download PS5 sdk sample (demo) is in the links of the tutorial https://youtu.be/LDPidynbElk

@Brutal-sam Do you mind re-uploading it please? Link's dead 😔 
And any other basic 3D graphics samples you may have 👀👌
